ngraph/python/CMakeLists.txt

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 # ******************************************************************************
 
-cmake_minimum_required (VERSION 3.1)
+cmake_minimum_required (VERSION 3.13)
```

Comment:
@ilyachur can this project be built out of main OpenVINO repository?

Reply:
Yes, my position is that we need to have only one entry point.